lld/unittests/CMakeLists.txt

```diff
@@ -4,5 +4,8 @@ function(add_lld_unittests test_dirname)
   add_unittest(LLDUnitTests ${test_dirname} ${ARGN})
 endfunction()
 
-add_subdirectory(AsLibAll)
-add_subdirectory(AsLibELF)
+# Temporary disable on Windows due to non-zero exit code returned by ld.lld
```

Comment:
* Please be especially careful about labelling our changes to upstream CMake files.  (Extra pity for potential future compiler engineers unfamiliar with both Windows and the intricacies of CMake…)
* Grammar nit (esp. for upstream)  s/b “Temporarily disable”.